### ISSUE TITLE
Check 2FA/SAML requirements for the bots

### ISSUE THREAD

**Issue by timvandermeij** [mozilla/pdf.js#14875]
On https://github.blog/2022-05-04-software-security-starts-with-the-developer-securing-developer-accounts-with-2fa/ I read that at the end of 2023 GitHub will require 2FA for all accounts, so that _could_ include our bot account.

On https://github.com/orgs/mozilla/teams/everybody-temp-comms/discussions/3 I noticed that enforced SAML _might_ possibly be a problem for our bots given that they explicitly mention robot accounts. I'm not sure if this holds given that the bots are probably not in the GitHub organization.

In short, we should figure out if the bots are impacted by these changes. If so, we need to find a way to deal with that, either by fixing the bot accounts or by moving to GitHub Actions faster (see #11851).

/cc @brendandahl @calixteman

**Comment by marco-c** [comment]
> On https://github.com/orgs/mozilla/teams/everybody-temp-comms/discussions/3 I noticed that enforced SAML might possibly be a problem for our bots given that they explicitly mention robot accounts. I'm not sure if this holds given that the bots are probably not in the GitHub organization.

We are indeed not affected by this because the bot user is an outside collaborator.

**Comment by timvandermeij** [comment]
Closing since we have determined any problems and we are in the process of moving away from the bots anyway in #11851.
